lib/longtail_platform.c

```diff
@@ -5,6 +5,7 @@
 
 static const uint32_t HostnamePrime = 0x01000193;
 static const uint32_t HostnameSeed  = 0x811C9DC5;
+static const uint32_t MaxChunkSize  = 16777216;
```

Comment:
Easier to read if `= 0x16 * 1024u * 1024u`

Reply:
You meant `16u * 1024u * 1024u`?

de5ac8e3d5ef7b157d6e97f6518ae1eb12599f46